### PR TITLE
cleanup(compute): Cleaning up code for Compute samples

### PR DESCRIPTION
**Pull request by m-strzelczyk**
Fixing multiple small issues common to many Compute samples:

1. Unnecessary `return`s at the end of functions.
2. Unused imports.
3. Incorrect usage of `NoReturn` type hint.
4. Any minor issue I found during this cleanup.

Please pay attention only to files in the `compute/client_library/ingredients/` folder, the rest is generated from the ingredients, so it just a lot of repeated changes.

### CODE REVIEW

File: compute/client_library/ingredients/disks/regional_delete.py [hunk 1]
```diff
@@ -16,14 +16,13 @@
 # folder for complete code samples that are ready to be used.
 # Disabling flake8 for the ingredients file, as it would fail F821 - undefined name check.
 # flake8: noqa
-import sys
 from typing import NoReturn
```
Comment (rsamborski) [inline compute/client_library/ingredients/disks/regional_delete.py:19]: Can be removed as well

**Review by rsamborski — APPROVED** [review]
Little nit, otherwise approved.

**Review by engelke — APPROVED** [review]

**Comment by kweinmeister** [comment]
```
Traceback (most recent call last):
  File "/workspace/compute/client_library/test_sgs.py", line 52, in test_snippets_freshness
    pytest.fail(
  File "/workspace/compute/client_library/.nox/py-3-8/lib/python3.8/site-packages/_pytest/outcomes.py", line 194, in fail
    raise Failed(msg=reason, pytrace=pytrace)
Failed: This test fails because file snippets/instances/change_machine_type.py seems to be outdated. Please run `python sgs.py generate` to update your snippets.
```

**Comment by leahecole** [comment]
rebuilding 2.7 because it's about to timeout

**Pull request merged**